src/components/Item/ItemForm.tsx

```diff
@@ -8,6 +8,7 @@ import { getDropAction, handlePaste } from '../Editor/helpers';
 import { MarkdownEditor, allowNewLine } from '../Editor/MarkdownEditor';
 import { c } from '../helpers';
 import { Item } from '../types';
+import { TFile } from "obsidian";
```

Comment:
I recommend using `'obsidian` rather than `"obsidian"`

Reply:
Hey, good catch on this ! TY for reviewing - Made the changes you brought up + fixed some formatting up